mantidproject/mantid
# Remove check for existence of recent files list on MantidPlot startup

**Description of work.**

If a filepath that appears in the recent files list in MantidPlot is on a mounted network drive and the network starts showing a big slowdown

**Report to:** [nobody]. 

**To test:**

I used a tool called [wondershaper](http://xmodulo.com/limit-network-bandwidth-linux.html) to limit the download/upload speed of my network interface to 10KB/s: `sudo wondershaper eno1 10 10`.  Before this fix MantidPlot sat for minutes before it started but now it starts as expected.

Remember to clear the restrictions after testing: `sudo wondershaper clear eno1`. Note that `eno1` is the name of the interface on my machine `netstat -i` will list the known interfaces.

Refs #12425 

---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/IndividualTicketTesting/)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
